ampd/src/event_processor.rs

```diff
@@ -105,64 +144,118 @@ mod tests {
     use crate::event_processor::{EventHandler, EventProcessor};
     use async_trait::async_trait;
     use error_stack::{Report, Result};
+    use futures::executor::block_on;
```

Comment:
many imports are unused.

Reply:
yeah it's not cleaned up yet, I just wanted your eyes on the test